app/models/miq_worker.rb

```diff
@@ -498,19 +518,9 @@ def self.nice_increment
     delta.kind_of?(Integer) ? delta.to_s : "+10"
   end
 
+  # TODO: Rename this!!!
```

Comment:
I like the way you think... Yes, once we don't have a command line, it's useless.